lib/form/paragraph3.flow

```diff
@@ -630,160 +674,155 @@ keepFormTogether(form : Form) -> bool {
 	};
 }
 
-ParaLineAcc(lines : List<Form>, width : double, y : double, baseline : double);
+ParaLineAcc(lines : List<Form>, y : double, baseline : double);
 
 // renders lines and returns resulting forms along with their tight width at the rendering time
 renderParaLines(
 	availableWidth : double,
 	lines : [ParaLine],
 	tightWidth : bool,
-	alignment: ParaLineAlignment,
+	alignment: GeneralAlignment,
 	interlineSpacing : double,
 	topLineBaseline : bool,
 	styleTree : Tree<int, [ParaElementInteractiveStyle]>,
 	rtl : bool,
 	isMarked : bool
-) -> Pair<Form, double> {
+) -> Triple<Form, double, double> {
```

Comment:
Even Pair is not clear. Tripple is really bad. do not use Tripple, make special data type instead

Reply:
Bad habits from untyped languages. Sure.